profiling/src/profiling/mod.rs

```diff
@@ -966,6 +972,21 @@ impl Profiler {
                 sample_values.extend_from_slice(&values[3..5]);
             }
 
+            #[cfg(php_has_fibers)]
+            {
+                let fiber = unsafe { ddog_php_prof_get_active_fiber() };
+                if !fiber.is_null() {
+                    if let Some(functionname) =
+                        unsafe { extract_function_name(&(*(*fiber).fci_cache.function_handler)) }
```

Comment:
@bwoebi Can you double-check the expectation here? Is the `fci_cache` always going to hold a pointer to a valid `zend_function`?

Reply:
Yes. The fcc is set by Fiber::__construct as part of zpp, which will always set the function_handler on success, and there's nothing changing that value in all of fibers afterwards, from start to destruction of the fiber itself.

Reply:
Thanks for checking 🙇